src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/PostgreSqlConnection.php

```diff
@@ -118,6 +118,8 @@ public function getExtraSetupSqlForTable(Table $createdTable): array
 
     private function getTriggerSql(): array
     {
+        $functionName = str_replace('.', '_', $this->configuration['table_name']);
+
```

Comment:
The PR fixes the bug, but still create the function inside the default schema (public)
I suggest to add this code in order to put the function in the specified schema

```suggestion
if (true === isset($schemaName)) {
            return [
                // create trigger function
                sprintf(<<<'SQL'
CREATE OR REPLACE FUNCTION %1$s.notify_%2$s() RETURNS TRIGGER AS $$
    BEGIN
        PERFORM pg_notify('%2$s', NEW.queue_name::text);
        RETURN NEW;
    END;
$$ LANGUAGE plpgsql;
SQL
                    ,$schemaName
                    ,$functionName),
                // register trigger
                sprintf('DROP TRIGGER IF EXISTS notify_trigger ON %s;', $this->configuration['table_name']),
                sprintf('CREATE TRIGGER notify_trigger AFTER INSERT OR UPDATE ON %1$s FOR EACH ROW EXECUTE PROCEDURE %2$s.notify_%3$s();', $this->configuration['table_name'], $schemaName, $functionName)
            ];
        }

Reply:
You are right take a look at my changes

Reply:
Great